triggerbot/tree_watcher.py

```diff
@@ -12,6 +12,8 @@
 
 from threading import Timer
 from collections import defaultdict
+from mozci.query_jobs import TreeherderApi
```

Comment:
When I try this locally I get ```ImportError: No module named query_jobs```, do we need a release of mozci before this can land?